.github/workflows/ci.yaml

```diff
@@ -0,0 +1,68 @@
+name: "CI"
```

Comment:
We are somewhat inconsistent with strings in YAML: sometimes it is double quotes, sometimes single, sometimes no quotes. I read that good practice is to use no quotes unless you need to use them (e.g. '10' so it is not parsed as number, or 'true' so it is not parsed as boolean, or special characters). I think in our case that would mean removing most of the quotes we are using so far.

Reply:
I've changed that too :)